src/imars3d/backend/diagnostics/rotation.py

```diff
@@ -85,7 +89,7 @@ def _find_rotation_center(
         atol = atol_deg if in_degrees else np.radians(atol_deg)
         idx_low, idx_hgh = find_180_deg_pairs_idx(angles, atol=atol, in_degrees=in_degrees)
         # process
-        max_workers = None if max_workers <= 0 else max_workers
+        max_workers = clamp_max_workers(max_workers)
```

Comment:
Thanks for capturing this one.
It was supposed to be updated in the last round of refactoring of max_worker.